### ISSUE TITLE
Siderophores

### ISSUE THREAD

**Issue by wwjvdsande** [OpenSourceMycetoma/What-other-molecules-to-screen#10]
We have recently performed and analysed the transcriptomic responses of _Madurella mycetomatis_ grain formation in our _Galleria mellonella_ infection model. Siderophores where expressed in all stadia of grain formation. See: https://www.biorxiv.org/content/10.1101/2022.12.20.520897v1

Since siderophores are important for the pathogen to obtain iron, they are most likely secreted, than obtain iron from the host and taken up by the fungus again to use the iron for their growth. This means that the siderophore can be taken up inside the grain. Therefore siderophores might be used as trojan horses to enter the fungal cell inside the grain. 

This concept has been exploited for Aspergillus fumigatus using the Aspergillus specific siderophore. GR-2397, a novel antifungal agent was based on the siderophore ferrichrome. A siderophore also produced by Madurella mycetomatis. 
![image](https://user-images.githubusercontent.com/36398138/235458120-a098b0f6-9af5-4511-a746-0c66e1201609.png)

@OpenSourceMycetoma/corecontrib Would it be possible to couple some of our fenarimols with excellent in vitro activity but no in vivo activity to ferrichrome to see if it can enter the grain in that way? That would be a cool solution to get molecules in the fungal cell?

**Comment by wwjvdsande** [comment]
Apparently coupling to ferrichrome has been done for antimalarial agents. So it should be possible: https://link.springer.com/article/10.1007/s10534-022-00375-8

**Comment by drc007** [comment]
It has also been used for anti-bacterials

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6767078/ <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6767078/>

Cheers

Chris

**Comment by bebi78** [comment]
@wwjvdsande @mattodd Yes, conjugation with penicillins is possible. So it should also be possible to prepare fenarimol conjugates, I think via the piperazine. I might ask some people of our former network in this field if OK (see Angewandte review article). Hydroxamate-based siderophores should be tried. 
https://onlinelibrary.wiley.com/doi/10.1002/anie.201701586
https://www.sciencedirect.com/science/article/abs/pii/S0040402006008544
https://link.springer.com/article/10.1007/BF00298377

**Comment by bebi78** [comment]
@JuliaSarju Would your colleague Dr. Duhme-Klair like to contribute to the siderophore project?

**Comment by dmitrij176** [comment]
![gr1_lrg](https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/assets/44785501/c066f12a-25f4-4091-807c-3687e6cb2220)

Further to the conversation that we had during last MycetOS meeting, I have a few questions regarding concept application.

Generally, there are 4 classical types of siderophore modifications which are usually performed in the related field:
1. Carboxylate
2. Phenolate 
3. Catecholate
4. Hydroxamate

If we consider modifying any of the already potent Fenarimols or other MYOS series from the existing library, the following questions arise:
1. Can the existing molecules be modified or would they have to be synthesised by different chemical routes?
2. How many siderophore groups should be added? Most known structures (both natural and synthetic) have multiple units incorporated, with mixed groups being common as well.
3. Which groups to add preferentially (say out of these 4 in the scheme)?

Any thoughts?
@mattodd @bebi78 @wwjvdsande

**Comment by wwjvdsande** [comment]
@dmitrij176 we will have most chance on success if we take the fungal siderophores. Madurella mycetomatis produces hydroxyferricrocin, ferricrocin and fusarinine C like aspergillus does, but not TAFC. TAFC is specific for Aspergillus. It is therefore most logical to take either hydroxyferricicrocin or fusarinine C.
![image](https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/assets/36398138/2737d7a9-4a86-4943-b500-034a6e4b80e5)

Molonoy was able to lable fusarinine C with an fluorescent label as demonstrated below:
![image](https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/assets/36398138/02998adf-9be6-4d8f-9ed6-f11bdd3f738a)

Tiwari and all used ferricrocin to couple antimalaria agents:
![image](https://github.com/OpenSourceMycetoma/What-other-molecules-to-screen/assets/36398138/996d2f9f-c163-40f2-bab7-b8f23448f481)

See: https://www.sciencedirect.com/science/article/pii/S1874391915302244?via%3Dihub#f0015 and https://link.springer.com/article/10.1007/s10534-022-00375-8

One siderophore per molecule should be enough I think. For these examples it was also enough to get it at least in Aspergillus. I could ask Sean Doyle of we can test the fluorescent one in Madurella hyphae. It fluorescences green so it will be difficult to test it in the grain itself (which also autofluorescences in green) but not impossible.

**Comment by wwjvdsande** [comment]
With the discussions on the Siderophore biosynthesis pathway in the last MycetOS meetings, I found two interesting papers. In one of the papers several molecules were found to dock in the active sites of SidA. Does anyone know what they are?

SMILES of the top 10 compounds from docking 
 

<img width="572" height="501" alt="Image" src="https://github.com/user-attachments/assets/3ccaae5c-08d2-4bcc-8d22-d2e502fd0d28" />

Papers are: 

[SidA expression in A fumigatus paper.pdf](https://github.com/user-attachments/files/23234393/SidA.expression.in.A.fumigatus.paper.pdf)
[SidA expression in A fumigatus paper truncated form - supplementary files.pdf](https://github.com/user-attachments/files/23234394/SidA.expression.in.A.fumigatus.paper.truncated.form.-.supplementary.files.pdf)
[SidA expression in A fumigatus paper truncated form.pdf](https://github.com/user-attachments/files/23234395/SidA.expression.in.A.fumigatus.paper.truncated.form.pdf)

**Comment by MFernflower** [comment]
two of the compounds listed in that image have pubchem entries and do not seem too hard to make or buy

https://pubchem.ncbi.nlm.nih.gov/compound/4360814

https://pubchem.ncbi.nlm.nih.gov/compound/4715058


@OpenSourceMycetoma/corecontrib

**Comment by mattodd** [comment]
I would be super hesitant about getting excited about molecules predicted to bind based on docking, period.

SidA is known to be inhibited by celastrol, but it's not a nice molecule.

https://pubmed.ncbi.nlm.nih.gov/27588426/

Let's get the protein made and screened!

**Comment by wwjvdsande** [comment]
@mattodd that is also the idea, but in these papers I can also check if there are special things to consider when expressing sidA. I noted that they did have some difficulties with the complete SidA in A. fumigatus but the first aminoacids seem to differ quite a lot for M. mycetomatis, so I will start with trying the full SidA protein first

**Comment by wwjvdsande** [comment]
@OpenSourceMycetoma/corecontrib a new paper on SidA appeared on pubmed: DOI: [10.1016/j.biochi.2026.06.008](https://doi.org/10.1016/j.biochi.2026.06.008)
